extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -273,6 +273,30 @@ abstract class RangerSparkExtensionSuite extends KyuubiFunSuite with SparkSessio
       doAs("admin", sql(s"DROP DATABASE IF EXISTS $db"))
     }
   }
+
+  test("show functions") {
+    val default = "default"
+    val db3 = "default3"
+    val function1 = "function1"
+    try {
+      doAs("admin", sql(s"CREATE FUNCTION $function1 AS 'Function1'"))
+      doAs("admin", assert(sql(s"show user functions $default.$function1").collect().length == 1))
+      doAs("bob", assert(sql(s"show user functions $default.$function1").collect().length == 0))
+
+      doAs("admin", sql(s"CREATE DATABASE IF NOT EXISTS $db3"))
+      doAs("admin", sql(s"CREATE FUNCTION $db3.$function1 AS 'Function1'"))
+
+      doAs("admin", assert(sql(s"show user functions $db3.$function1").collect().length == 1))
+      doAs("bob", assert(sql(s"show user functions $db3.$function1").collect().length == 0))
+
+      doAs("admin", assert(sql(s"show system functions").collect().length > 0))
+      doAs("bob", assert(sql(s"show system functions").collect().length === 0))
```

Comment:
the builtin functions shall have open ACLs

Reply:
updated, the builtin functions do't check.